sc/shared/src/test/scala/sigma/LanguageSpecificationV6.scala

```diff
@@ -1611,4 +1611,94 @@ class LanguageSpecificationV6 extends LanguageSpecificationBase { suite =>
 
   }
 
+  property("Coll.reverse") {
+    val f = newFeature[Coll[Int], Coll[Int]](
+      { (xs: Coll[Int]) => xs.reverse },
+      """{(xs: Coll[Int]) => xs.reverse }""".stripMargin,
+      sinceVersion = VersionContext.V6SoftForkVersion
+    )
+
+    verifyCases(
+      Seq(
+        Coll(1, 2) -> Expected(ExpectedResult(Success(Coll(2, 1)), None)),
+        Coll[Int]() -> Expected(ExpectedResult(Success(Coll[Int]()), None))
+      ),
+      f
+    )
+  }
+
+  property("Coll.distinct") {
+    val f = newFeature[Coll[Int], Coll[Int]](
+      { (xs: Coll[Int]) => xs.distinct },
+      """{(xs: Coll[Int]) => xs.distinct }""".stripMargin,
+      sinceVersion = VersionContext.V6SoftForkVersion
+    )
+
+    verifyCases(
+      Seq(
+        Coll(1, 2) -> Expected(ExpectedResult(Success(Coll(1, 2)), None)),
+        Coll(1, 1, 2) -> Expected(ExpectedResult(Success(Coll(1, 2)), None)),
+        Coll(1, 2, 2) -> Expected(ExpectedResult(Success(Coll(1, 2)), None)),
```

Comment:
Suggest to add a case with unordered collections and many different duplicates.

Reply:
added